### PR TITLE
Added Artifact_Examples

### PR DESCRIPTION
**Pull request by augmetio**
## Requirements

Workflow contributors, verify you have completed the following items by checking them off:

### Checklist
- [x] Workflow verified as working in a test
- [x] Workflow bundle includes `Name_of_Workflow.icon` file, `help.md` file, `workflow.spec.yaml` file, and `screenshots` directory
- [x] Workflow title should follow general [Title of Articles](http://grammar.yourdictionary.com/capitalization/rules-for-capitalization-in-titles.html) format
- [x] Workflow includes a detailed Description
- [x] Workflow step names follow [Title of Articles](http://grammar.yourdictionary.com/capitalization/rules-for-capitalization-in-titles.html) format
- [x] Triggers, Decisions, and other workflow steps include descriptions where needed
- [x] Workflow includes at least one `Final Report` artifact that summarizes workflow execution details
- [x] Workflow bundle (directory) is named after the title of the workflow in snake case with first letters capitalized, e.g. `workflows/Name_of_Workflow/`
- [x] `help.md` file includes Description, Key Features, Requirements, Setup, Technical Details, Version History, and References
- [x] `workflow.spec.yaml` file includes `products`, `name`, `title`, `description`, `use_cases` tags, `keywords` tags, `source_url`, and the `name` for each screenshot matches a PNG image in the `screenshots` directory

### Assessment

Post your results with screenshots in this PR using the following style:

```
### Screenshots

Screenshots help your reviewers provide valuable feedback on your contribution. To add a screenshot here, simply drag and drop the PNG files from a file explorer.

#### Workflow Builder

![Artifact_Examples_Workflow](https://user-images.githubusercontent.com/35856445/80739627-c6c1fd00-8ae4-11ea-930e-35719033ec72.png)

#### Job

<Insert Job Image Here>

#### Artifact

![Artifact_Examples_Job_First_Artifact](https://user-images.githubusercontent.com/35856445/80739665-d3deec00-8ae4-11ea-891a-ebd560cc8e10.png)
![Artifact_Examples_Job_Second_Artifact](https://user-images.githubusercontent.com/35856445/80739677-d7727300-8ae4-11ea-921a-40ab9f557092.png)

```

### CODE REVIEW

File: workflows/Artifact_Examples/help.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+# Version History
+
+* 1.0.0 - Initial workflow
+
+# Links
+
+## References
+
+* https://github.com/rapid7/insightconnect-workflows
```
Comment (tylerslijboom-r7) [inline workflows/Artifact_Examples/help.md:37]:
```suggestion
* [InsightConnect Workflows](https://github.com/rapid7/insightconnect-workflows)
```

File: workflows/Artifact_Examples/help.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+# Key Features
+
+* Examples of how to present data in Artifacts
+* No Plugins or Orchestrator needed, good for training
+* All in the Cloud so execution is fast
+
+# Requirements
+
+none
```
Comment (tylerslijboom-r7) [inline workflows/Artifact_Examples/help.md:13]:
```suggestion
None
```

File: workflows/Artifact_Examples/help.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+# Documentation
+
+## Setup
+
+Once the workflow has been downloaded, login to InsightConnect and "Import" it into the workflow builder. There are intentionally no connections to configure and does not need an Orchestrator because no Plugins are used.
+
+## Technical Details
+
+Plugins utilized by workflow: none
```
Comment (tylerslijboom-r7) [inline workflows/Artifact_Examples/help.md:23]:
```suggestion
|Plugin|Version|Count|
|----|----|--------|
```

File: workflows/Artifact_Examples/workflow.spec.yaml [hunk 4]
```diff
@@ -0,0 +1,23 @@
+extension: workflow
+products: ["insightconnect"]
+name: Artifact_Examples
+title: "Artifact Examples"
+description: "This uses an API Trigger with each of the DataTypes. The workflow then has different Actions to see how to work with that data"
```
Comment (tylerslijboom-r7) [inline workflows/Artifact_Examples/workflow.spec.yaml:5]:
```suggestion
description: "This uses an API Trigger with each of the DataTypes. The workflow then has different Actions to see how to work with that data."
```

File: workflows/Artifact_Examples/workflow.spec.yaml [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+hub_tags:
+  use_cases: [alerting_and_notifications, data_enrichment]
+  keywords: [example,artifact,enrichment]
+  features: []
+resources:
+  source_url: https://github.com/rapid7/insightconnect-workflows/tree/master/workflows/Artifact_Examples
+  license_url: https://github.com/rapid7/insightconnect-workflows/blob/master/LICENSE
+  screenshots:
+  - name: Artifact_Examples_Job_First_Artifact.png
+    title: Example of presenting data directly from the Trigger
```
Comment (tylerslijboom-r7) [inline workflows/Artifact_Examples/workflow.spec.yaml:19]:
```suggestion
    title: Example of Presenting data directly from the Trigger
```

File: workflows/Artifact_Examples/workflow.spec.yaml [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+  - name: Artifact_Examples_Job_Second_Artifact.png
+    title: Example of presenting data inside a Loop
```
Comment (tylerslijboom-r7) [inline workflows/Artifact_Examples/workflow.spec.yaml:21]:
```suggestion
    title: Example of Presenting data inside a Loop
```

**Comment by augmetio** [comment]
Thanks, this was my first one so I was guessing on some of those fields from other examples

File: workflows/Artifact_Examples/workflow.spec.yaml [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+products: ["insightconnect"]
+name: Artifact_Examples
+title: "Artifact Examples"
+description: "This uses an API Trigger with each of the DataTypes. The workflow then has different Actions to see how to work with that data"
+version: 1.0.0
+vendor: community
+support: community
+status: []
+hub_tags:
+  use_cases: [alerting_and_notifications, data_enrichment]
```
Comment (jschipp-r7) [inline workflows/Artifact_Examples/workflow.spec.yaml:11]:
```suggestion
  use_cases: [data_utility]
```

File: workflows/Artifact_Examples/workflow.spec.yaml [hunk 8]
```diff
@@ -0,0 +1,23 @@
...
+  keywords: [example,artifact,enrichment]
```
Comment (jschipp-r7) [inline workflows/Artifact_Examples/workflow.spec.yaml:12]:
```suggestion
  keywords: [example]
```

File: workflows/Artifact_Examples/help.md [hunk 9]
```diff
@@ -0,0 +1,37 @@
+# Description
+
+This uses an API Trigger with each of the DataTypes. The workflow then has different Actions to see how to work with that data.
```
Comment (jschipp-r7) [inline workflows/Artifact_Examples/help.md:3]:
```suggestion
This workflow lets you explore how to use various data types within InsightConnect. It uses an API Trigger for manual input for each of the available types and then runs different actions to show how they work and finally prints a resulting artifact card displaying the values.
```

**Comment by augmetio** [comment]
no problem, done

**Comment by jschipp-r7** [comment]
Released https://extensions.rapid7.com/extension/Artifact_Examples

**Comment by jschipp-r7** [comment]
Hey @augmetio, we found a bug in the endpoint for `community` contributed workflows. We have hidden the workflow for the time being until we fix the bug preventing user's from successfully importing it. There's a PR up for this now, and we plan to have it resolved soon.

**Comment by augmetio** [comment]
What was it?



-------- Original message --------

**Comment by jschipp-r7** [comment]
@augmetio we had an issue where workflows vendored as `community` instead of `rapid7` would not import correctly directly from the Extension Library. That is now fixed!

https://extensions.rapid7.com/extension/Artifact_Examples

Screen Shot 2020-05-18 at 5 06 19 PM
Screen Shot 2020-05-18 at 5 06 12 PM

**Pull request merged**